config-tab.js

```diff
@@ -1,56 +1,58 @@
 // Config tab initialization and handlers
-import { updateStatusIndicator, syncToArchiveBox, getArchiveBoxServerUrl } from './utils.js';
+import { updateStatusIndicator, getArchiveBoxServerUrl, addToArchiveBox } from './utils.js';
 
 export async function initializeConfigTab() {
   const configForm = document.getElementById('configForm');
   const serverUrl = document.getElementById('archivebox_server_url');
   const apiKey = document.getElementById('archivebox_api_key');
   const matchUrls = document.getElementById('match_urls');
   const excludeUrls = document.getElementById('exclude_urls');
-  
+
   // Load saved values
   const archivebox_server_url = await getArchiveBoxServerUrl();
-  const { archivebox_api_key, match_urls, exclude_urls } = await chrome.storage.local.get([
+  const { archivebox_api_key, match_urls, exclude_urls, enable_auto_archive } = await chrome.storage.local.get([
     'archivebox_api_key',
     'match_urls',
     'exclude_urls',
+    'enable_auto_archive',
   ]);
-  console.log('Got config values from storage:', archivebox_server_url, archivebox_api_key, match_urls, exclude_urls);
+  console.log('Got config values from storage:', archivebox_server_url, archivebox_api_key, match_urls, exclude_urls, enable_auto_archive);
 
   // migrate old config_archiveboxBaseUrl to archivebox_server_url
   const {config_archiveBoxBaseUrl} = await chrome.storage.sync.get('config_archiveboxBaseUrl', );
   if (config_archiveBoxBaseUrl) {
     await chrome.storage.local.set({ archivebox_server_url: config_archiveBoxBaseUrl });
   }
-  
+
   serverUrl.value = archivebox_server_url || '';
   apiKey.value = archivebox_api_key || '';
   matchUrls.value = typeof match_urls === 'string' ? match_urls : '';
   excludeUrls.value = typeof exclude_urls === 'string' ? exclude_urls : '';
 
+  // Set the auto-archive toggle state
+  const enableAutoArchive = document.getElementById('enable_auto_archive');
```

Comment:
more descriptive variable name pls, this is an element reference not a bool

`const enable_auto_archive_elem = ...`